addons/crm/static/src/scss/crm_lead_views.scss

```diff
@@ -0,0 +1,8 @@
+// Lead kanban view related stuff
+.o_kanban_view.o_opportunity_kanban {
+    .o_kanban_counter .o_kanban_counter_progress {
+        &.o_crm_kanban_counter_progress_mrr {
+            width: 50%;
```

Comment:
Can't we instead have flex or col in the dom (and tweak the original template by extension, in addition to adding the recurring revenue strong element) ? Purpose would be to try to remove the dynamic play of classes and css to have only templates if possible.

Reply:
Hello @tde-banana-odoo ,

We already use flex (in the existing design) 🙂 As far as the custom CSS is concerned, we've removed the file, and is now managed with the class on the element when MRR is enabled 🙂